includes/functions/functions_lookups.php

```diff
@@ -336,7 +336,7 @@ function zen_requires_attribute_selection($products_id) {
               left join " . TABLE_PRODUCTS_OPTIONS . " po on pa.options_id = po.products_options_id
               where pa.products_id = " . (int)$products_id . "
               and po.language_id = " . (int)$_SESSION['languages_id'] . "
-              group by pa.options_id, pa.products_attributes_id";
+              group by pa.options_id";
```

Comment:
Also: the comments in this commit may help if you want to add an undocumented parameter to your `configure.php` to force a certain SQL mode for testing:
https://github.com/zencart/zencart/commit/f456c766bb69575dce47b2776463400b2c42f4ac